k8s/logdna-agent.yaml

```diff
@@ -121,6 +121,16 @@ spec:
                   key: logdna-agent-key
             - name: RUST_LOG
               value: info
+            - name: LOGDNA_LOG_K8S_EVENTS
+              value: always
```

Comment:
Should this be on by default?

Reply:
Not sure, should clarify during our meeting today